charts/prism/values.yaml

```
@@ -682,3 +697,6 @@ deployments:
   - computations
   - invitations
   - ui
+
+env:
+  prod: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)